apps/src/lib/kits/maker/CircuitPlaygroundBoard.js

```diff
@@ -117,6 +122,14 @@ export default class CircuitPlaygroundBoard extends EventEmitter {
       this.fiveBoard_.io.reset();
     }
     this.fiveBoard_ = null;
+
+    // Deregister Firmata sysex response handler for circuit playground commands,
+    // or playground-io will fail to register a new one next time we construct it
+    // and the old playground-io instance will get events.
+    if (Firmata.SYSEX_RESPONSE) {
+      delete Firmata.SYSEX_RESPONSE[CP_COMMAND];
+    }
+    delete Playground.hasRegisteredSysexResponse;
```

Comment:
Here's the fix for problem 2: On destroy, we clear playground-io's handler for incoming data out of Firmata's static handler list, and tell playground-io that it's okay to re-register next time it's constructed.  This makes me :cry: because it means we can't construct more than one `CircuitPlaygroundBoard` at a time and have them work, but in practice that's a nonissue for now.